apps/verify/models/read-cookie.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const dataService = require('../../common/data-service');
+const {
+makeRequest,
```

Comment:
bad indentation

Reply:
good spot copy pasted from elsewhere, I've fixed that as well now 😸